ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/system-control-plane.j2

```diff
@@ -0,0 +1,36 @@
+{% if system.control_plane is arista.avd.defined %}
+
+## System Control-Plane
+{%     if system.control_plane.tcp_mss.ipv4 is arista.avd.defined or system.control_plane.tcp_mss.ipv6 is arista.avd.defined %}
+
+#### TCP MSS Ceiling
+
+| Protocol | Segment Size |
+| -------- | -------------|
+{%         if system.control_plane.tcp_mss.ipv4 is arista.avd.defined %}| ipv4 | {{ system.control_plane.tcp_mss.ipv4 }} |
```

Comment:
Please break the line after the`%}` to make it more readable. It will not add newlines to generated output. same for ipv6

Reply:
Done in 7f64921e53cc2146c20a104a9fb2105c2ad1d79d

In addition, I've also updated the subsequent section in similar style
```jinja2
#### Control-Plane Access-Groups

| Protocol | VRF | Access-list |
| -------- | --- | ------------|
{#         IPv4 Access-groups #}
{%         for acl_set in system.control_plane.ipv4_access_groups | arista.avd.natural_sort %}
| IPv4 | {{ acl_set.vrf | arista.avd.default('default') }} | {{ acl_set.acl_name }} |
{%         endfor %}
{#         IPv6 Access-groups #}
{%         for acl_set in system.control_plane.ipv6_access_groups | arista.avd.natural_sort %}
| IPv6 | {{ acl_set.vrf | arista.avd.default('default') }} | {{ acl_set.acl_name }} |
{%         endfor %}
{%     endif %}
```